tools/oesign/main.c

```diff
@@ -147,9 +159,17 @@ int dump_parser(int argc, const char* argv[])
         oe_err("--enclave-image option is missing");
         ret = 1;
     }
+
     if (!ret)
+    {
         /* dump oeinfo and signature information */
-        ret = oedump(enclave);
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+        if (with_eeid)
+            ret = oedump_eeid(enclave);
+        else
+#endif
+            ret = oedump(enclave);
```

Comment:
Fix the indentation as:
```
#ifdef OE_WITH_EXPERIMENTAL_EEID
        if (with_eeid)
            ret = oedump_eeid(enclave);
#endif
        ret = oedump(enclave);
```

Reply:
The indentation and the `else` are exactly as intended here. Removing the `else` will run both, `oedump_eeid` and `oedump` instead of just one of them.

Reply:
Actually, I changed my mind. I will change it as it's useful to get both at the same time anyways.

Reply:
(Done.)